automation_tools/__init__.py

```diff
@@ -338,6 +338,10 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
         proxy = 'capsule'
     else:
         proxy = 'foreman-proxy'
+    if os.environ.get('SATELLITE_VERSION') in ('6.0', '6.1', '6.2', '6.3'):
```

Comment:
Think opposite way here:
the option stayed same for all these versions and changed for nightly which is not listed 
(nightly,6.4, 6.5)

(I know that due to rebase to 1.15 probably the value '6.3' would need to be removed later on)